src/components/Link.js

```diff
@@ -31,6 +31,7 @@ const ExternalLink = styled.a`
 `
 
 const InternalLink = styled(IntlLink)`
+  white-space: nowrap;
```

Comment:
Unfortunately this will affect all other internal links, and force them to not wrap at all when they should word-wrap.